test/models/auxiliar_business_unit_test.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AuxiliarBusinessUnitTest < ActiveSupport::TestCase
+  test 'remove all scored businessUnit after destroy' do
+    plan_item_one     = plan_items :current_plan_item_1
+    business_unit_two = business_units :business_unit_two
+
+    auxiliar_business_unit               = AuxiliarBusinessUnit.new
+    auxiliar_business_unit.plan_item     = plan_item_one
+    auxiliar_business_unit.business_unit = business_unit_two
+
+    plan_item_one.auxiliar_business_units << auxiliar_business_unit
+    plan_item_one.save
+
+    control_objective_item = control_objective_items :management_dependency_item
+
+    control_objective_item.scored_business_unit = business_unit_two
+    control_objective_item.save
```

Comment:
Acá siempre conviene llamarlo así:
```suggestion
    control_objective_item.save!
```
Sino puede fallar una validación y _pasa_ sin notarse.

Reply:
franco aca tengo un temita. Resulta que cuando hago el save! de auxiliar_business_unit, me funciona bien. Ahora si hago del control_objective_item me tira una excepcion de una validacion que ya se emitio el informe y no puedo hacerlo. Le hice un update_atribute con auxiliar_bussines_unit_id y que skipee la validacion. Porque en los fixtures la verdad no encontre uno que lo pueda modificar. Generar un fixture de una control_objective_item al no tener todas las reglas de negocio en la mente nose con que datos ponerlo para poder guardarlo.

Reply:
Ahh claro, no me di cuenta que estabas usando uno "no editable". Fijate con `:management_dependency_item_editable`, ese debería funcionar sin problemas.

Reply:
ahora si, justo ese no lo habia probado